client/my-sites/email/email-management/home/email-mailbox-action-menu.jsx

```diff
@@ -93,10 +94,15 @@ const getTitanMenuItems = ( {
 	translate,
 } ) => {
 	const email = getEmailAddress( mailbox );
+	const isEmbeddedMailboxEnabled = isEnabled( 'emails/embedded-inbox-testing' );
+
+	const mailboxUrl = isEmbeddedMailboxEnabled
```

Comment:
This should still be named a prefix because it still only forms the beginning part of the url.